analysis_options.yaml

```diff
@@ -1,6 +1,7 @@
 include: package:pedantic/analysis_options.1.8.0.yaml
 
 analyzer:
+  exclude: [test_resources/**]
```

Comment:
Why we don't want to have beautiful test projects :P ?

Reply:
I'll try to check how I can fix the analysis error that occurs there. It seemed like something related to imports but the project actually works if you try to run it from its directory :/.